src/pages/AddItem/components/ProductImg/ProductImg.jsx

```diff
@@ -49,7 +49,7 @@ function ProductImg ({ name, value, onChange }) {
             <label className='title'>상품 이미지</label>
             <div className='img-box'>
                 
-                <div className='custom-button' onClick={handleButtonClick}>
+                <label className='custom-button' onClick={handleButtonClick}>
```

Comment:
P1:
label 태그를 쓰셨으니 onClick 대신 htmlFor로 input과 연결하세요~

```suggestion
                <label className='custom-button' htmlFor={inputId}>
```